### PR TITLE
feat(sbb-autocomplete): introduce size s

### LINKED ISSUE #2768: Autocomplete: Size S
https://www.figma.com/file/9r6xSfNmEfCFxl1yFYedrj/Lyne-Components?type=design&node-id=29421%3A256678&mode=design&t=6s3eVNfR5eROI3dH-1

### PR DESCRIPTION
**Pull request by DavideMininni-Fincons**
## Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [pull request tracker](https://github.com/sbb-design-systems/lyne-components/pulls) for a Pull Request (PR) that matches the one I want to submit, without success.

## Issue

This PR Closes #2768 

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)

See [Review Guidelines](../docs/REVIEW.md) for more information what is checked during review process.

## Changes

Changes in this pull request:

- introduce examples for sbb-autocomplete in form-field size=s

## Browsers

I tested the build on the following browsers:

- [ ] Firefox Desktop
- [x] Chrome Desktop
- [ ] Edge Desktop
- [ ] Safari Desktop
- [ ] Chrome Mobile
- [ ] Safari Mobile

## Screen readers

I tested the build on the following browsers:

- [ ] JAWS Firefox Desktop
- [ ] JAWS Chrome Desktop
- [ ] NVDA Firefox Desktop
- [ ] NVDA Chrome Desktop
- [ ] VoiceOver Safari Desktop
- [ ] VoiceOver Chrome Desktop
- [ ] VoiceOver Safari Mobile
- [ ] Android Accessibility Suite Chrome Mobile

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by TomMenga — COMMENTED** [review]
Let's wait 'till Monday for the answer to the stories topic

Ps. Do you plan to adapt the autocomplete grid too?

File: src/elements/autocomplete-grid/autocomplete-grid/autocomplete-grid.visual.spec.ts [hunk 1]
```diff
@@ -150,7 +151,7 @@ describe('sbb-autocomplete-grid', () => {
       for (const borderless of [false, true]) {
```
Comment (jeripeierSBB) [inline src/elements/autocomplete-grid/autocomplete-grid/autocomplete-grid.visual.spec.ts:151]: Somehow I like it more if directly after the for loop there is a describe() setting the name there. With that not every it() needs e.g. to include negative=, borderless=, ...  what do you think? (would also be the case for the non grid variant)

**Review by jeripeierSBB — APPROVED** [review]
LGTM, one possible improvement

File: src/elements/autocomplete-grid/autocomplete-grid/autocomplete-grid.visual.spec.ts (hunk 1 above)
Comment (DavideMininni-Fincons) [reply src/elements/autocomplete-grid/autocomplete-grid/autocomplete-grid.visual.spec.ts]: Fix done! i went for a maximum of 2 'states' in each 'it' block

Comment (jeripeierSBB) [reply src/elements/autocomplete-grid/autocomplete-grid/autocomplete-grid.visual.spec.ts]: thank you :-)

**Comment by mcilurzo** [comment]
LGTM. Thanks

**Pull request merged**